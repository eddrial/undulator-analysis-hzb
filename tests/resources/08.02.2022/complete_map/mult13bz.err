 error report for the data set mult13bz.dvm        
--------------------------------------------
      
         127   data points thrown away to get a
         monotonically increasing data set
 the truncated data set has only   1591 data points 
        out of totally   2558 data points
 the derivative at  0.43065E+03 and  0.48178E+03 is zero 
 the derivative at  0.48178E+03 and  0.53967E+03 is zero 
 the derivative at  0.53967E+03 and  0.62164E+03 is zero 
 the derivative at  0.64995E+03 and  0.70319E+03 is zero 
 the derivative at  0.70319E+03 and  0.73403E+03 is zero 
 the derivative at  0.76150E+03 and  0.84305E+03 is zero 
 the derivative at  0.90009E+03 and  0.95629E+03 is zero 
 the derivative at  0.95629E+03 and  0.98756E+03 is zero 
 the derivative at  0.98756E+03 and  0.10404E+04 is zero 
 the derivative at  0.10404E+04 and  0.10949E+04 is zero 
 the derivative at  0.11236E+04 and  0.11807E+04 is zero 
 the derivative at  0.11807E+04 and  0.12639E+04 is zero 
 the derivative at  0.13205E+04 and  0.13776E+04 is zero 
 the derivative at  0.13776E+04 and  0.14338E+04 is zero 
 the derivative at  0.14621E+04 and  0.15200E+04 is zero 
 the derivative at  0.15200E+04 and  0.16564E+04 is zero 
 the derivative at  0.18259E+04 and  0.18854E+04 is zero 
 the derivative at  0.18854E+04 and  0.19378E+04 is zero 
 the derivative at  0.19649E+04 and  0.20190E+04 is zero 
 the derivative at  0.20464E+04 and  0.20773E+04 is zero 
 the rms value of the residuals is  0.20412E-02
 the maximum residual is  0.53261E-02
 the rms value of the lambda/2-filtered data is  0.15045E-02
 the maximum value of the lambda/2-filtered data is  0.98774E-02
