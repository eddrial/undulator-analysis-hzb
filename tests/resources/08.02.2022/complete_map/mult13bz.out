mult13bz.dvm
            

 names of parameters as follows : 
b(1) == amplitude, 1st harm.
b(2) == amplitude, 3rd harm.
b(3) == amplitude, 5rd harm.
....                        
b(n) == amplitude, n^th harm.
b(n+1) == lambda, mm.
b(n+2) == x0, 1st harm, mm.
b(n+3) == x0, 3rd harm, mm.
b(n+4) == x0, 5th harm, mm.
....
b(nparm) == x0, n^th harm, mm.

 parameters:
   1 -7.6255370E-03
   2 -1.1796338E-04
   3  6.4282227E-05
   4 -5.8097821E-06
   5 -5.7187317E-06
   6  5.6031494E+01
   7  1.0410525E+03
   8  1.0116155E+03
   9  1.0445667E+03
  10 -6.4091247E-01
  11 -1.0007809E+00
 number of iterations:   13
     sumsq:  6.62874E-03
       rms:  2.04118E-03  dimension of magnetic field
       rms:  2.67672E+01  percent of b_eff
   rms_ext:  1.30880E+01  percent of b_eff
 B_eff =   7.6256E-03
 B_max =  -7.5454E-03
 K_eff =    0.040
 lamda_1 (nm) =      2.54230

        eigenvectors in parameter space

 error                         direction
    15.93,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00
    12.59,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00
     2.01,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00
     1.83,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00
     0.10,  0.00  0.00  0.00  0.00  0.00 -0.05  1.00  0.00  0.00  0.00  0.00
     0.01,  0.00  0.00  0.00  0.00  0.00  1.00  0.05  0.00  0.00  0.00  0.00
     0.00, -0.23 -0.35  0.67  0.33 -0.51  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.84 -0.16  0.23 -0.40 -0.22  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.34 -0.46 -0.49  0.66 -0.06  0.00  0.00  0.00  0.00  0.00  0.00
     0.00, -0.06 -0.61  0.28 -0.12  0.73  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.35  0.52  0.42  0.53  0.39  0.00  0.00  0.00  0.00  0.00  0.00
   
  calculated kicks for trajectory compensation: 
 kick_1:   0.19042 Tmm
 kick_2:  -0.19042 Tmm
      
 the rms value of the second derivative is  0.00000E+00
 the maximum value of the second derivative is  0.00000E+00
      
 the maximum transverse movement of the electron in mu_meter
                  (peak to peak) is  0.48176E+01
      
 the truncated data set has only   1591 data points 
      
 the rms value of the residuals is  0.20412E-02
 the maximum residual is  0.53261E-02
      
 the rms value of the lambda/2-filtered data is  0.15045E-02
 the maximum value of the lambda/2-filtered data is  0.98774E-02
      
 the rms phase error (all data) is    1.809 deg.
 the maximum phase error (all data) is    3.437 deg.
      
 the rms phase error (extrema) is    6.176 deg.
 the maximum phase error (extrema) is   12.554 deg.
      
 the Walker-rms phase error (all data) is    1.809 deg.
 the Walker-maximum phase error (all data) is    3.483 deg.
      
 the Walker-rms phase error (extrema) is    1.819 deg.
 the Walker-maximum phase error (extrema) is    3.396 deg.
  
  phase: straight line subtracted
 -----------------------------------------
 total phase error             1.808739    
 local phase error            0.5070508    
 slowly varying phase error  NaN           
 oscillatory phase error     NaN           
 -----------------------------------------
